core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,14 +154,14 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
```

Comment:
With the latest commit, in case a None is returned in describeGroup(), an exception is thrown so at that point I don't think that case will need to be handled here. But if we had stuck to a println, I think it would have needed something here. What do you think?